build-tools-internal/src/main/groovy/elasticsearch.auto-backporting.gradle

```diff
@@ -36,6 +36,7 @@ allprojects {
 
     dependencies {
         rewrite "org.openrewrite:rewrite-java-11"
+        rewrite "com.breskeby.rewrite:java-recipes:1.0.0"
```

Comment:
Can we avoid having this live in your personal repo? Any reason those implementations can't live build tools, or perhaps and elastic-owned repository?

Reply:
the main reason was that the original recipes test support is kotlin based. I'll check if I can port that to groovy with not to much hassle. a seperate elastic-owned repo would work of course.

Reply:
@mark-vieira I've moved the custom rules into build-tools-internal. our func test coverage should be enough for now. The test support rewrite provides is heavy relying on Kotlin and I do not want to include kotlin code to the codebase just for these tests